pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -16,7 +16,7 @@ in mkDerivation {
 
   src = fetchurl {
     url = "https://protonmail.com/download/protonmail-bridge_${version}_amd64.deb";
-    sha256 = "032ggk9fvd19fbsqkzwzwh0hpyg8gpkrin71di7zsx6ias5innw1";
+    sha256 = "34fdc917fb2c6f06ec6c6a71041f31ad3599b4ee5556e09ad919c36d71eac247";
```

Comment:
Is this correct? I got `0iy2x9qnvhqrv6df0mjmxss9jddd64gh8wbadkn0cvrczcbwkz9l`

Reply:
The one I input used a sha256, not the nix-prefix.
Might be worth updating the version as well, as I think there has been a later update, by this point.

Reply:
Cheers @privateseabass hadn't realized that was an option.

Reply:
No worries.
Most people hadn't.